.circleci/config.yml

```diff
@@ -0,0 +1,13 @@
+version: 2.1
+jobs:
+  build:
+    working_directory: ~/ah-frontend
+    docker:
```

Comment:
Are we using `docker` in our project, or it is a must to include that in the config for `circle ci`?

Reply:
Docker is a virtual machine used by circle CI to run our test